deploy/openshift/clusterrole.yaml

```diff
@@ -20,6 +20,11 @@ rules:
   - get
   - list
   - watch
+- apiGroups: ["route.openshift.io"]
```

Comment:
there is a "routes" in the "" apiGroup - I think we need to delete that, no?

Reply:
These same changes need to be made to the operator cluster role templates for the kiali and kiali-viewer roles... so make these same changes here:

* https://github.com/kiali/kiali/blob/master/operator/roles/kiali/templates/openshift/clusterrole.yaml
* https://github.com/kiali/kiali/blob/master/operator/roles/kiali/templates/openshift/clusterrole-viewer.yaml

Keep in mind, this deploy/clusterrole.yaml file (all of these under deploy/) are going away in favor of the operator. When I merge my operator PR these will be deleted. So we need to remember to keep the operator templates up to date going forward.

Reply:
actually, @mwringe - I will add these to the operator PR right now. That way it will be in the same PR with everything else. so never mind my last comment :)

Reply:
Here's the commit containing the changes to the operator that incorporate the new cluster role data: https://github.com/kiali/kiali/pull/974/commits/b56fe93311d0a7d6c4a75020b87e67e5229dad92

Reply:
> there is a "routes" in the "" apiGroup - I think we need to delete that, no?

I left it in there to make sure it wont affect anything when running on 3.x. Leaving it in doesn't cause any problems, but we could likely remove it.